src/PointwiseFunctions/GeneralRelativity/ProjectionOperators.hpp

```diff
@@ -0,0 +1,83 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
```

Comment:
you only need `TypeAliases.hpp` in the header file, then include `Tensor.hpp` in the `cpp` file. Type aliases basically is a bunch of forward declarations :)

Reply:
thanks for catching this, fixed.